helper/helper.go

```diff
@@ -212,7 +212,20 @@ func preloadCreatedPipesForRestore(oidWithBatchList []oidWithBatch, queuedPipeCo
 	}
 }
 
-func getOidWithBatchListFromFile(oidFileName string) ([]oidWithBatch, error) {
+type IHelper interface {
+	getOidWithBatchListFromFile(oidFileName string) ([]oidWithBatch, error)
+	createPipe(pipe string) error
+	flushAndCloseRestoreWriter(pipeName string, oid int) error
+	closeAndDeletePipe(tableOid int, batchNum int)
+}
+
+type Helper struct{}
+
+func (h Helper) createPipe(pipe string) error {
+	return createPipe(pipe)
+}
+
+func (h Helper) getOidWithBatchListFromFile(oidFileName string) ([]oidWithBatch, error) {
```

Comment:
I think this function should be moved to `helper/restore_helper.go`, because it is only used there.